src/os_auth/local-server.c

```diff
@@ -319,54 +341,36 @@ cJSON* local_add(const char *id, const char *name, const char *ip, char *groups,
     }
 
     // Check for duplicated ID
-
     if (id && (index = OS_IsAllowedID(&keys, id), index >= 0)) {
-        if (force >= 0 && (antiquity = OS_AgentAntiquity(keys.keyentries[index]->name, keys.keyentries[index]->ip->ip), antiquity >= force || antiquity < 0)) {
-            id_exist = keys.keyentries[index]->id;
-            minfo("Duplicated ID '%s' (%s). Removing old agent.", id, id_exist);
-            add_remove(keys.keyentries[index]);
-            OS_DeleteKey(&keys, id_exist, 0);
-        } else {
+        minfo("Duplicated ID '%s'.", keys.keyentries[index]->id);
```

Comment:
Same typo here: "duplicate ID".

Reply:
Fixed. This change implicated revision in other code sections where the same typo is present.